extensions-core/datasketches/src/test/java/org/apache/druid/query/aggregation/datasketches/theta/sql/ThetaSketchSqlAggregatorTest.java

```diff
@@ -1073,6 +1081,38 @@ public void testThetaSketchEstimateAsVirtualColumn()
     );
   }
 
+  @Test
+  public void testThetaSketchEstimateAsVirtualColumn2Args()
+  {
+    testQuery(
+        "SELECT"
+        + " THETA_SKETCH_ESTIMATE(thetasketch_dim3, FALSE)"
+        + " FROM foo",
```

Comment:
I don't expect this table to actually have enough items to generate numbers that would need rounding.  You should be able to access the wikipedia data set in the tests, which you should be able to get to estimate things.

Note that the smallest logk for Theta sketch allowed is 4, which is 16 entries.  So, you would have to do a theta sketch aggregation with a size of 16 and then see more than 16 distinct values in order to get it to estimate (and generate a true floating point number)

Reply:
Thanks, I am using a bigger dataset for the `true` case will update shortly